test/jasmine/tests/treemap_test.js

```diff
@@ -1796,6 +1796,78 @@ describe('treemap uniformtext', function() {
             fontsizes: [13, 13, 13, 13, 13, 13, 13, 13, 13, 13, 13],
             scales: [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
         }))
+        .then(function() {
+            fig.layout.uniformtext = undefined; // back to default
+            return Plotly.react(gd, fig);
+        })
+        .then(assertTextSizes('clear uniformtext', {
+            fontsizes: [12, 12, 12, 12, 12, 12, 12, 12, 12, 12, 12],
+            scales: [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0.84],
+        }))
+        .catch(failTest)
+        .then(done);
+    });
+
+    it('should uniform text scales after transition', function(done) {
```

Comment:
Similarly here:

```
Chrome 78.0.3904 (Linux 0.0.0) treemap uniformtext should uniform text scales after transition FAILED
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 0', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 1', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 2', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 4', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 5', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0, 1, 'scale for element 7', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 9', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 10', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.86 to be close to 0.9883899233296827, 1, 'scale for element 0', 'after click child'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.86 to be close to 0.9883899233296827, 1, 'scale for element 1', 'after click child'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.86 to be close to 0.9883899233296827, 1, 'scale for element 2', 'after click child'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.86 to be close to 0.9883899233296827, 1, 'scale for element 3', 'after click child'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.86 to be close to 0.9883899233296827, 1, 'scale for element 4', 'after click child'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.86 to be close to 0.9883899233296827, 1, 'scale for element 5', 'after click child'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.86 to be close to 0.7058823529411765, 1, 'scale for element 6', 'after click child'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 0', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 1', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 2', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 4', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 5', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0, 1, 'scale for element 7', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 9', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
	Error: Expected 0.84 to be close to 0.9424364123159304, 1, 'scale for element 10', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/treemap_test.js:1728:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:228778:40
```

Reply:
It should be improved in https://github.com/plotly/plotly.js/pull/4444/commits/2cf7defe563cbdaac46d880e6e54aa6f21965888.

Reply:
Nicely done. All tests pass now :ok_hand: